content/docs/user-guide/project-structure/dvcyaml-files.md

```diff
@@ -12,14 +13,113 @@ modify, write, or generate them by your own means.
 `dvc.yaml` files are designed to be small enough so you can easily version them
 with Git along with other <abbr>DVC files</abbr> and your project's code.
 
-[dependency graph]:
-  /doc/user-guide/pipelines/defining-pipelines#directed-acyclic-graph-dag
+## Metrics
+
+The list of `metrics` contains one or more paths to <abbr>metrics</abbr> files.
+Here's an example:
+
+```yaml
+metrics:
+  - metrics.json
+```
+
+Metrics are key/value pairs saved in structured files that map a metric name to
+a numeric value. See `dvc metrics` for more information and how to compare among
+experiments.
```

Comment:
I don't know if we should be pedantic, but metrics don't necessarily have to be key/value pairs. For example, `metrics.json` could just be:
```json
3.14
```

```console
$ dvc metrics show
Path                                                                  
metrics.json  3.14
```

They could be list too, but that seems unsupported at the moment (there's no reason not to support them though)
```yaml
- avg_prec: 0.9249974999612706
  roc_auc: 0.9460213440787918
- avg_prec: 0.9249974999612706
  roc_auc: 0.9460213440787918 
```

Reply:
🤔 That's new to me. I'm struggling to imagine how it's useful without a key. How would you show the list example in `metrics show`?